v2/dtos/common/metrics.go

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// MetricsRequest defines the request associated with the /metrics endpoint.
+// This object and its properties correspond to the MetricsRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/MetricsRequest
+type MetricsRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
+}
+
+// MetricsResponse defines the providing memory and cpu utilization stats of the service.
+// This object and its properties correspond to the MetricsResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/MetricsResponse
+type MetricsResponse struct {
+	BaseResponse   `json:",inline"`
+	MemAlloc       int  `json:"memAlloc"`
```

Comment:
shouldnt these be uint64?

Reply:
Updated `MemAlloc` and other fields as well, thanks.